app/routers/dayview.py

```diff
@@ -74,14 +79,23 @@ def _get_time_format(self) -> str:
         for multiday in [self.start_multiday, self.end_multiday]:
             yield self.MULTIDAY_FORMAT if multiday else self.DEFAULT_FORMAT
 
-    def _set_total_time(self) -> None:
+    def _set_total_time(self) -> str:
         length = self.end_time - self.start_time
         self.length = length.seconds / 60
         format_gen = self._get_time_format()
         start_time_str = self.start_time.strftime(next(format_gen))
         end_time_str = self.end_time.strftime(next(format_gen))
         return ' '.join([start_time_str, '-', end_time_str])
 
+    def _set_total_time_visiblity(self) -> bool:
+        return not self.length <= 60
+
+    def _set_title_size(self) -> Union[str, None]:
+        i = bisect_left(self.LENGTH_SIZE_STEP, self.length)
+        print(i)
+        if i < len(self.CLASS_SIZES):
```

Comment:
Can `i` somehow be large or equal to it?

Reply:
yes becouse if thr function gets number bigger then the last parmeter it will create an index error... it will give 3.